index.html

```diff
@@ -617,9 +576,20 @@ <h3><i class="fas fa-download"></i> Download</h3>
     <img class="sidebarIcons" src="images/Papirus-Team-Papirus-Apps-Amazon-store.svg">Get it from Amazon App Store
    </a>
 
+  <script src="javascripts/testimony.js" type="text/javascript"></script>
+  <h4 id="reviewHeading">User Reviews</h4>
+   <div id="reviewFirst"></div>
+   <div id="reviewSecond"></div>
+   <div id="reviewThird"></div>
+   <div id="reviewFourth"></div>
+   <div id="reviewFifth"></div>
+
+
```

Comment:
why are these two times? Also js file is loaded twice.

Reply:
The two renderings are for two different screen width contexts. The one in the sidebar only renders for browsers with a width above 900px I think. For lower widths, the sidebar is hidden. So in order for users with screen width less than 900px to still see the review function, I placed it again in a <div id="smScreen"> that only shows if the screen is less than 900px wide

Reply:
I see, ok